playbook/app/pb_kits/playbook/pb_button/_button.scss

```diff
@@ -1,7 +1,24 @@
 @import "./button_mixins";
 
+$pb_button_sizes: (
+  "sm":   0.75rem,
+  "md":   0.875rem,
+  "lg":   1.125rem,
+);
```

Comment:
In relation to my last comment, you could keep `"large"` etc here so that it doesn't break old code.

Reply:
This PR contains another PR and any changes to the button kit is not part of this PR.

Reply:
Same here

Reply:
@haydenrou Fixed